packages/tfm-python38/tfm-python38.spec

```diff
@@ -0,0 +1,144 @@
+%global scl_prefix_python %{scl_python}-
+
+%global install_scl 1
+
+# Do not produce empty debuginfo package.
+%global debug_package %{nil}
+
+Summary: Package that installs %scl
+Name: %scl_name
+Version: 1.0
+Release: 1%{?dist}
+License: GPLv2+
+Source1: README
+Source2: LICENSE
+BuildRequires: help2man
+BuildRequires: scl-utils-build
+BuildRequires: %{scl_prefix_python}scldevel
+BuildRequires: %{scl_prefix_python}python-devel
+%if 0%{?install_scl}
+Requires: %{scl_python}
+%endif
+
+%description
+This is the main package for %scl Software Collection.
+
+%package runtime
+Summary: Package that handles %scl Software Collection.
+Requires: scl-utils
+Requires: %{scl_prefix_python}runtime
+
+%description runtime
+Package shipping essential scripts to work with %scl Software Collection.
+
+%package build
+Summary: Package shipping basic build configuration
+Requires: scl-utils-build
+Requires: %{scl_runtime}
+Requires: %{scl_prefix_python}scldevel
+
+%description build
+Package shipping essential configuration macros to build %scl Software Collection.
+
+%package scldevel
+Summary: Package shipping development files for %scl
+
+%description scldevel
+Package shipping development files, especially useful for development of
+packages depending on %scl Software Collection.
+
+%package python3-devel
+Summary: Package pulling in %{scl_prefix_python}python-devel
+Requires: %{scl_prefix_python}python-devel
+
+%description python3-devel
+Package pulling in %{scl_prefix_python}python-devel
+
+%package python3-setuptools
+Summary: Package pulling in %{scl_prefix_python}python-setuptools
+Requires: %{scl_prefix_python}python-setuptools
+
+%description python3-setuptools
+Package pulling in %{scl_prefix_python}python-setuptools
+
+%prep
+%setup -T -c
+
+# This section generates README file from a template and creates man page
+# from that file, expanding RPM macros in the template file.
+cat >README <<'EOF'
+%{expand:%(cat %{SOURCE1})}
+EOF
+
+# copy the license file so %%files section sees it
+cp %{SOURCE2} .
+
+%build
+# generate a helper script that will be used by help2man
+cat >h2m_helper <<'EOF'
+#!/bin/bash
+[ "$1" == "--version" ] && echo "%{scl_name} %{version} Software Collection" || cat README
+EOF
+chmod a+x h2m_helper
+
+# generate the man page
+help2man -N --section 7 ./h2m_helper -o %{scl_name}.7
+
+%install
+rm -rf %{buildroot}
+mkdir -p %{buildroot}%{_scl_scripts}/root
+mkdir -p %{buildroot}%{_root_prefix}/lib/rpm/redhat
+cat >> %{buildroot}%{_scl_scripts}/enable << EOF
+export PATH=%{_bindir}\${PATH:+:\${PATH}}
+export LD_LIBRARY_PATH=%{_libdir}\${LD_LIBRARY_PATH:+:\${LD_LIBRARY_PATH}}
+export MANPATH=%{_mandir}:\${MANPATH}
+EOF
+%scl_install
+
+cat >> %{buildroot}%{_scl_scripts}/enable << EOF
+. scl_source enable %{scl_python}
+EOF
+
+# Create the scldevel subpackage macros
+
+cat >> %{buildroot}%{_root_sysconfdir}/rpm/macros.%{scl_name_base}-scldevel << EOF
+%%scl_%{scl_name_base} %{scl}
+%%scl_prefix_%{scl_name_base} %{scl_prefix}
```

Comment:
this produces warnings:
```
warning: Macro %scl_tfm needs whitespace before body
warning: Macro %scl_prefix_tfm needs whitespace before body
```

seems it doesn't like the `-` in the `scl_name_base`